deployment/aws-terraform/all-in-one/main.tf

```diff
@@ -1,48 +0,0 @@
-terraform {
```

Comment:
Why we remove main.tf? 
Please follow terraform standard structure https://www.terraform.io/language/modules/develop/structure

Reply:
Done!